### PR TITLE
Fix#3005 : Remove EDI History jquery ui

### LINKED ISSUE #3005: Remove jquery-ui from the Fees->'EDI History' module
Remove jquery-ui from the Fees->'EDI History' module

### PR DESCRIPTION
**Pull request by stu01509**
Fixes #3005

#### Short description of what this resolves:
Remove EDI History jquery ui

image

image

image

image

image

### CODE REVIEW

**Comment by stu01509** [comment]
Hello @bradymiller  @stephenwaite  @tywrenn 

Do you have any ideas or suggestions about these sections? Especially the table seems like used jquery data tables-UI.

image

image

image

**Comment by tywrenn** [comment]
@stu01509 I attempted to fix this up in my pull request that is now basically dead, #2927. If you use some of what I used in cleaning it up that might help you out.

**Comment by stu01509** [comment]
Hello @bradymiller @stephenwaite  @tywrenn 

Finally, It looks better than jquery UI pretty. I also remove datatales-jq to datatables-bs,
Please review it, Thanks :)

image

image

image

image

image

File: interface/billing/edih_view.php [hunk 1]
```diff
@@ -29,276 +29,302 @@
 <html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
 
 <head>
-    <title><?php echo xlt("edi history"); ?></title>
+    <title><?php echo xlt("EDI History"); ?></title>
     <meta http-equiv="content-type" content="text/html;charset=utf-8" />
 
-    <!-- TODO: Address no_bootstrap here !-->
-    <?php Header::setupHeader(['no_main-theme', 'no_bootstrap', 'datetime-picker', 'datatables', 'datatables-jqui', 'datatables-jqui-theme', 'datatables-scroller', 'datatables-scroller-jqui-theme', 'jquery-ui', 'jquery-ui-sunny']); ?>
+    <?php Header::setupHeader(['no_main-theme', 'datetime-picker', 'datatables', 'datatables-bs', 'datatables-scroller']); ?>
```
Comment (bradymiller) [inline interface/billing/edih_view.php:35]: rec dropping the `no_main-theme` and seeing what happens

Comment (bradymiller) [reply interface/billing/edih_view.php:35]: prob also need this token in there `datatables-dt`, which is generally needed whenever using `datatables-bs`

Comment (stu01509) [reply interface/billing/edih_view.php]: Hi @bradymiller sir

I removed the no_main-theme, and it seems like to apply your custom style. e.g. button color. 
Any suggestions or feedback?

image

**Comment by bradymiller** [comment]
Overall gui appears to look good.

Noted following error in console when opened tab:
image

Also noted following error in console at Notes->Open (enter note) -> Save
image

After get those errors sorted out, I think @stephenwaite may be able to test it out.

**Comment by stu01509** [comment]
Hi @bradymiller sir

Thanks for your review, I updated the commit, please review it again 😃.

**Comment by bradymiller** [comment]
hi @stu01509 , Looks like getting close. Noted this js error at EDI file->Submit:

image

**Comment by stu01509** [comment]
Hello @bradymiller 

Updated the commit :)

**Comment by stephenwaite** [comment]
hi @stu01509 , looks like there's a little issue with the edi file
Screenshot from 2020-04-27 09-03-49

**Comment by stu01509** [comment]
Hello @stephenwaite 

Thanks for your review, can you give the test file?
I will try to do completely testing :)

**Comment by stephenwaite** [comment]
sure, it's attached, thanks @stu01509 
[EDI_837_5010_Usecase1.txt](https://github.com/openemr/openemr/files/4540002/EDI_837_5010_Usecase1.txt)

**Comment by stu01509** [comment]
Hello @stephenwaite 

I reproduce in my environment, it seems good to work. I guess you deploy old commit version, because I already remove the all jquery-ui accordion. Please take look :)

image

**Comment by tywrenn** [comment]
@stu01509 Offending code:
` $('#x12rsp > #accordion').accordion({`

**Comment by stu01509** [comment]
Hello @stephenwaite and @tywrenn 

Thanks for your review and help, I updated the commit, please take look.
Thank you :)

**Comment by stu01509** [comment]
🏓 @stephenwaite

**Comment by stephenwaite** [comment]
hi @stu01509, am seeing
```
* e303d8786 - (HEAD, stu01509/fix-#3005-remove-jquery-ui) fix: remove unnecessary code (3 days ago) <Cliff Su>
* a4e3d4b15 - (fix-#3005-remove-jquery-ui) fix: jqeury syantax error (4 days ago) <Cliff Su>
* 0026ba715 - fix: remove no_main-theme and add datatables-dt (4 days ago) <Cliff Su>

```
as last 3 commits, did you push the update from 2 days ago? thank you

**Comment by stu01509** [comment]
Hi @stephenwaite 

Yes, I fixed the accordion issue in e303d8786 commit.

**Comment by stephenwaite** [comment]
ok, testing well, thank you @stu01509

**Comment by stu01509** [comment]
Thanks for your review and help 😁 @stephenwaite

**Comment by stephenwaite** [comment]
another openemr improvement from @stu01509 !
external-content duckduckgo com

**Comment by sjpadgett** [comment]
Aww come on! now I have to go out and find me an animation!:)

**Comment by stephenwaite** [comment]
how about 
external-content duckduckgo com

**Comment by sjpadgett** [comment]
A little ho-hum for my big personality don't ya think?

**Comment by stephenwaite** [comment]
external-content duckduckgo com

**Comment by im-Amitto** [comment]
@stu01509

**Comment by stu01509** [comment]
Let us say goodbye to outdated animation haha 😄😄

**Pull request merged**